doc/man3/ASN1_STRING_TABLE_add.pod

```diff
@@ -0,0 +1,56 @@
+=pod
+
+=head1 NAME
+
+ASN1_STRING_TABLE_add, ASN1_STRING_TABLE_get,
+ASN1_STRING_TABLE_cleanup - ASN1_STRING_TABLE manipulation functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/asn1.h>
+
+ int ASN1_STRING_TABLE_add(int nid, long minsize, long maxsize,
+                           unsigned long mask, unsigned long flags);
+ ASN1_STRING_TABLE * ASN1_STRING_TABLE_get(int nid);
+ void ASN1_STRING_TABLE_cleanup(void);
+
+=head1 DESCRIPTION
+
+ASN1_STRING_TABLE_add() adds a new B<ASN1_STRING_TABLE> item into the
```

Comment:
I *think* it would be valuable to describe the type itself as well, maybe add a few words on what it's for.  If you need inspiration on how to structure that, have a look at any manual where the synopsis contains a typedef (I wrote the OSSL_STORE pages in man3 with this in mind, for example)